src/index.ts

```diff
@@ -381,6 +388,45 @@ app.get('/api/signing-nonce', (req, res) => {
   }
 });
 
+app.get(
```

Comment:
It's better to create a `/api/evm/**` subroute, instead of appending `evm-*` to each endpoint. Also, it can be moved to a separate router so that we don't add more clutter to this file

Like it was done in `routers/slise-ad-rules`